manuscript/book.md

```diff
@@ -319,7 +319,7 @@ Start thinking about:
 
 ## Avoid conditions
 
-Conditions make code harder to read and harder to test. They add nesting and make lines of code longer, so you have to split them into several lines. Each condition doubles the minimum amount of test cases you need to write for a certain module of function.
+Conditions make code harder to read and harder to test. They add nesting and make lines of code longer, so you have to split them into several lines. Each nested condition doubles the minimum amount of test cases you need to write for a certain module or function.
```

Comment:
Yeah, that's correct but doesn't fit the narrative because I'm talking about any conditions, not just nested ones. So let's change the other word ;-)

```suggestion
Conditions make code harder to read and harder to test. They add nesting and make lines of code longer, so you have to split them into several lines. Each condition increases the minimum number of test cases you need to write for a certain module or function.
```